compute/cker/include/cker/Types.h

```diff
@@ -145,6 +145,7 @@ struct ConvParams
   int32_t input_offset;
   int32_t weights_offset;
   int32_t output_offset;
+  // TODO: output_multiplier and output_shift can be multiple for per-channel quantized weight
   int32_t output_multiplier;
```

Comment:
- Channel sizes are:
```
T(0:1) UINT8 (32, 3, 3, 3) B(2) MobilenetV1/Conv2d_0/weights
T(0:7) UINT8 (64, 1, 1, 32) B(8) MobilenetV1/Conv2d_1_pointwise/weights
T(0:13) UINT8 (128, 1, 1, 64) B(14) MobilenetV1/Conv2d_2_pointwise/weights
T(0:19) UINT8 (128, 1, 1, 128) B(20) MobilenetV1/Conv2d_3_pointwise/weights
T(0:25) UINT8 (256, 1, 1, 128) B(26) MobilenetV1/Conv2d_4_pointwise/weights
T(0:37) UINT8 (512, 1, 1, 256) B(38) MobilenetV1/Conv2d_6_pointwise/weights
T(0:43) UINT8 (512, 1, 1, 512) B(44) MobilenetV1/Conv2d_7_pointwise/weights
T(0:49) UINT8 (512, 1, 1, 512) B(50) MobilenetV1/Conv2d_8_pointwise/weights
T(0:55) UINT8 (512, 1, 1, 512) B(56) MobilenetV1/Conv2d_9_pointwise/weights
T(0:61) UINT8 (512, 1, 1, 512) B(62) MobilenetV1/Conv2d_10_pointwise/weights
T(0:67) UINT8 (512, 1, 1, 512) B(68) MobilenetV1/Conv2d_11_pointwise/weights
T(0:73) UINT8 (1024, 1, 1, 512) B(74) MobilenetV1/Conv2d_12_pointwise/weights
T(0:79) UINT8 (1024, 1, 1, 1024) B(80) MobilenetV1/Conv2d_13_pointwise/weights
T(0:83) UINT8 (1001, 1, 1, 1024) B(84) MobilenetV1/Logits/Conv2d_1c_1x1/weights
```
- I will not modify `ConvParams`, But I will reuse int8 per-channel implementation for multipllier nad output_shift.